File: back_api/acessibilityguide/src/main/java/com/acessibilityguide/api/model/MarkerType.java
package com.acessibilityguide.api.model;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

import com.avaje.ebean.Model;

@Entity
@Table(name = "marker_type")
public class MarkerType extends Model {

	@Id
    private Long id;

    private String name;
    
    private String iconName;

    
	public String getIconName() {
		return iconName;
	}

	public void setIconName(String iconName) {
		this.iconName = iconName;
	}

	public Long getId() {
		return id;
	}

	public void setId(Long id) {
		this.id = id;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}
    
	
}

File: back_api/acessibilityguide_back/src/main/kotlin/com/acessibilityguide/api/model/MarkerType.kt
package com.acessibilityguide.api.model

import com.avaje.ebean.Model
import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table

/**
 * Created by eduardolopes on 02/06/17.
 */

@Entity
@Table(name = "marker_type")
class MarkerType: Model() {

    @Id
    var id: Long? = null

    var name: String? = null

    companion object : Model.Find<Long, MarkerType>() { }
}